cdap-ui/app/cdap/components/EntityCard/ProgramMetrics/index.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, {Component, PropTypes} from 'react';
+import {MyProgramApi} from 'api/program';
+import {default as NamespaceStore} from 'services/store/store';
+
+export default class ProgramMetrics extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      status: '',
+      numRuns: 0,
+      appName: this.props.entity.applicationId,
+      loading: true
+    };
+  }
+
+  componentWillMount() {
+    const apiProgramTypeConvert = {
```

Comment:
We could probably have this as a constant. I am sure we will be using the same thing in logs, status and other APIs too.